core/vm/instructions_zkevm.go

```diff
@@ -147,8 +148,16 @@ func opSendAll_zkevm(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContex
 	return nil, errStopToken
 }
 
+func makeLog_zkevm_logIndexFromZero(size int) executionFunc {
+	return makeLog_zkevm(size, true)
+}
+
+func makeLog_zkevm_regularLogIndexes(size int) executionFunc {
+	return makeLog_zkevm(size, false)
+}
+
 // [zkEvm] log data length must be a multiple of 32, if not - fill 0 at the end until it is
-func makeLog_zkevm(size int) executionFunc {
+func makeLog_zkevm(size int, logInexShouldBeFromZero bool) executionFunc {
```

Comment:
Small typo - I wonder if it should be logIndexPerTx  - true if we index0 per tx, or false if we index0 per block?